src/Microsoft.OData.Core/ODataBatchWriter.cs

```diff
@@ -4,6 +4,8 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using Microsoft.OData.Core.JsonLight;

```

Comment:
This namespace is not (and should not) be used by this base class, so this line should be deleted.